components/post/default.htm

```diff
@@ -18,11 +18,13 @@
 
 <p class="info">
     {% if post.categories.count %}
-        {% set categoryLinks = post.categories | map(c => "<a href=\"#{c.url}\">#{c.name}</a>") | join(', ') %}
-
+        {% set categoryLinks = [] %}
+        {% for category in post.categories %}
+            {% set categoryLinks = categoryLinks|merge(["<a href=\"#{category.url}\">#{category.name}</a>"]) %}
+        {% endfor %}
```

Comment:
I think it might just be better to go straight back to https://github.com/rainlab/blog-plugin/blame/35be2bae4a5e0a3506057061cf00cbc6fe36cd2e/components/posts/default.htm#L11-L13 but just wrapping it in a `{% set categoryLinks %} {% endset %}` tag.

Reply:
I'll take a look at it and update my MR when I have time.